operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -1676,8 +1709,60 @@ def getSSLOrNone(server):
   except:
     trace("Ignoring getSSL() exception, this is expected.")
     ret = None
```

Comment:
This 'except' looks like dead-code -  getSSLOrNone or getSSLPortIfEnabled should be handling the exception.
It should be removed in case it mistakenly captures some other exception...

Reply:
merged into getSSLPortIfEnabled, I removed the trace statement also.